src/app/features/admin/cqc-main-service-change-list/cqc-main-service-change-list.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds govuk-!-margin-bottom-5">
+    <h2 class="govuk-heading-m">CQC main service change</h2>
+  </div>
+  <div class="govuk-grid-column-two-thirds">
+    <table class="govuk-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header">Workplace</th>
+          <th scope="col" class="govuk-table__header">Received</th>
+          <th scope="col" class="govuk-table__header">Status</th>
+        </tr>
+      </thead>
+      <tbody class="govuk-table__body">
+        <tr class="govuk-table__row" *ngFor="let cqcServiceChange of pendingCQCMainServiceChanges">
+          <td class="govuk-table__cell">
+            <a
+              [attr.data-testid]="'workplaceName-' + cqcServiceChange.orgName"
+              [routerLink]="['/', cqcServiceChange.establishmentUid]"
```

Comment:
Should this link be prefixed with a `/`? 
```suggestion
              [routerLink]="[cqcServiceChange.establishmentUid]"
```